polaris-react/src/components/Navigation/components/Section/Section.tsx

```diff
@@ -139,16 +139,20 @@ export function Section({
 
   const toggleRollup = rollup && items.length > rollup.after && (
     <div className={styles.ListItem} key="List Item">
-      <button
-        type="button"
-        className={toggleClassName}
-        onClick={toggleExpanded}
-        aria-label={ariaLabel}
-      >
-        <span className={styles.Icon}>
-          <Icon source={HorizontalDotsMinor} />
-        </span>
-      </button>
+      <div className={styles.ItemWrapper}>
+        <div className={styles.ItemInnerWrapper}>
```

Comment:
What does this change do? Doesn't seem like it needs to be related to the nav item highlight.

Reply:
I see.. seems like overkill though since now we're surrounding one element with 3 wrapping divs.

Reply:
Yeah I had that feeling too, but the alternative is adding these styles directly into the `.RollupToggle` class, which I thought felt a bit bespoke but can be done

Reply:
This toggle is "trying" to be a nav item but not quite so its a bit awkward

Reply:
I don't feel too strongly but rollup being its own thing would be alright too if we can share spacing variables.

Reply:
You know what though, it does need the containers to have the same background hover as the other items, the first container constraining space then the second applying hover colouring etc and then it handles its content inside that.